release/models/bgp/openconfig-bgp-types.yang

```diff
@@ -713,23 +722,24 @@ module openconfig-bgp-types {
   typedef community-type {
     type enumeration {
       enum STANDARD {
-        description "Send only standard communities";
+        description "Send standard communities";
       }
       enum EXTENDED {
-        description "Send only extended communities";
+        description "Send extended communities";
       }
-      enum BOTH {
```

Comment:
You will need to leave BOTH as an enum to keep this a non-breaking change.  You should add `status deprecated;` to this enum